kinto/core/storage/postgresql/__init__.py

```diff
@@ -703,7 +703,8 @@ def _format_conditions(self, filters, id_field, modified_field,
                 sql_field = "coalesce({}, '')".format(column_name)
                 # Cast when comparing to number (eg. '4' < '12')
                 try:
-                    if is_numeric(value) or all([is_numeric(v) for v in value]):
+                    # all([]) is True while all(None) raises a TypeError.
+                    if is_numeric(value) or all([is_numeric(v) for v in value] or None):
```

Comment:
Why not 👍  :) But we could also write `if value and (is_numeric(value) or all([...]))` no?